spec/dor/workflow_object_spec.rb

```diff
@@ -22,10 +22,8 @@
       @item.workflowDefinition.content = '<workflow-def id="accessionWF"/>'
     end
 
-    it 'indexes the number of archived objects for the workflow' do
-      expect(Dor::Config.workflow.client).to receive(:count_archived_for_workflow).and_return(5)
-      expect(@item.to_solr).to include 'accessionWF_archived_isi' => 5
+    it 'indexes the workflow name' do
+      expect(@item.to_solr).to include 'workflow_name_ssim' => ['accessionWF']
     end
   end
```

Comment:
i believe the parent provides a to_solr method (the original calls `super solr_doc, *args`).  i think essentially all dor-services objects have to_solr methods, courtesy of... `ActiveFedora::Base`?  

the tests pass with this call (and weren't previously failing w/ a method missing error, it was a mismatch in actual vs expected field value).